tests/functional/disaster-recovery/sc_arbiter/test_zone_unaware_app.py

```diff
@@ -0,0 +1,346 @@
+
+    @pytest.fixture()
+    def init_sanity(self, request, nodes):
+        """
+        Initial Cluster sanity
+        """
+        self.sanity_helpers = Sanity()
+
+        def finalizer():
+            """
+            Make sure all the nodes are Running and
+            the ceph health is OK at the end of the test
+            """
+
+            # check if all the nodes are Running
+            log.info("Checking if all the nodes are READY")
+            master_nodes = get_nodes(node_type=constants.MASTER_MACHINE)
+            worker_nodes = get_nodes(node_type=constants.WORKER_MACHINE)
+            nodes_not_ready = list()
+            nodes_not_ready.extend(
+                [node for node in worker_nodes if node.status() != "Ready"]
+            )
+            nodes_not_ready.extend(
+                [node for node in master_nodes if node.status() != "Ready"]
+            )
+
+            if len(nodes_not_ready) != 0:
+                try:
+                    nodes.start_nodes(nodes=nodes_not_ready)
+                except Exception:
+                    log.error(
+                        f"Something went wrong while starting the nodes {nodes_not_ready}!"
+                    )
+                    raise
+
+                retry(
+                    (
+                        CommandFailed,
+                        TimeoutError,
+                        AssertionError,
+                        ResourceWrongStatusException,
+                    ),
+                    tries=30,
+                    delay=15,
+                )(wait_for_nodes_status(timeout=1800))
+                log.info(
+                    f"Following nodes {nodes_not_ready} were NOT READY, are now in READY state"
+                )
+            else:
+                log.info("All nodes are READY")
+
+            # check cluster health
+            try:
+                log.info("Making sure ceph health is OK")
+                self.sanity_helpers.health_check(tries=50, cluster_check=False)
+            except CephHealthException as e:
+                assert (
+                    "HEALTH_WARN" in e.args[0]
+                ), f"Ignoring Ceph health warnings: {e.args[0]}"
+                get_ceph_tools_pod().exec_ceph_cmd(ceph_cmd="ceph crash archive-all")
+                log.info("Archived ceph crash!")
+
+        request.addfinalizer(finalizer)
+
+    @pytest.mark.parametrize(
+        argnames="fencing",
+        argvalues=[
+            pytest.param(
+                True,
+            ),
+            pytest.param(
+                False,
+            ),
+        ],
+        ids=[
+            "With-Fencing",
+            "Without-Fencing",
+        ],
+    )
+    def test_zone_shutdowns(
+        self,
+        init_sanity,
+        setup_logwriter_cephfs_workload_factory,
+        setup_logwriter_rbd_workload_factory,
+        logreader_workload_factory,
+        setup_network_fence_class,
+        nodes,
+        fencing,
+    ):
+
```

Comment:
can you please add test steps here? It will be easy to check what this test does.

Reply:
Done